src/yardstick/store/tool.py

```diff
@@ -10,15 +10,26 @@
 RESULT_SET_DIR = os.path.join("result", "sets")
 
 
+def install_base(name: str, store_root: str | None = None) -> str:
+    if not store_root:
+        store_root = store_config.get().store_root
+
+    return os.path.join(store_config.get().store_root, TOOL_DIR, name.replace("/", "_"))
+
+
 def install_path(config: artifact.ScanConfiguration, store_root: str | None = None) -> str:
     if not store_root:
         store_root = store_config.get().store_root
 
+    store_base = install_base(name=config.tool_name, store_root=store_root)
+
+    version = config.tool_version
+    if config.tool_name.startswith("grype"):
```

Comment:
The `tool` base class knowing about grype is another signal that it might be time to refactor.

Reply:
one of a hundred signals 🔥